pkg/ebpf/common.go

```diff
@@ -32,6 +32,9 @@ var (
 	ErrNotImplemented = errors.New("BPF-based system probe not implemented on non-linux systems")
 
 	nativeEndian binary.ByteOrder
+	// newAPIKernelVersion indicates the minimum version of kernel that uses the new API signature
+	// the affected APIs are kprobe/tcp_sendmsg, kprobe/udp_sendmsg, kprobe/udp_recvmsg
+	newAPIKernelVersion = "4.1.0"
```

Comment:
You don't need this as a constant to be honest - lets just move this comment to the function `hasOldKernelAPI` and rename it